### PR TITLE
add ONNX support for LeViT

### PR DESCRIPTION
**Pull request by gcheron**
# What does this PR do?

This PR adds ONNX support for LeViT. Linked to [#16308](https://github.com/huggingface/transformers/issues/16308).

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: tests/onnx/test_onnx_v2.py [hunk 1]
```diff
@@ -196,6 +196,7 @@ def test_values_override(self):
     ("xlm-roberta", "xlm-roberta-base"),
     ("layoutlm", "microsoft/layoutlm-base-uncased"),
     ("layoutlmv3", "microsoft/layoutlmv3-base"),
+    ("levit", "facebook/levit-128S"),
```
Comment (regisss) [inline tests/onnx/test_onnx_v2.py:199]: Have you checked if this test passes?

**Review by regisss — APPROVED** [review]
Thanks for adding this @gcheron!!

File: tests/onnx/test_onnx_v2.py (hunk 1 above)
Comment (gcheron) [reply tests/onnx/test_onnx_v2.py:199]: Yes, it does.

**Comment by regisss** [comment]
Pinging @lewtun and @sgugger for approval.
One CI test failed but it doesn't seem to be related to this PR.

**Review by sgugger — APPROVED** [review]
Good for me, @lewtun you can merge if you're satisfied.
Thanks a lot for the PR!

**Review by lewtun — APPROVED** [review]
Thanks for adding ONNX support for this architecture @gcheron !

The code looks good and before I merge can you confirm whether the slow tests pass, e.g. running:

```
RUN_SLOW=1 pytest tests/onnx/test_onnx_v2.py -k "levit"
```

Thanks!

**Comment by gcheron** [comment]
You are welcome, thank you for your work!
Yes, following the doc I have already run this command and it passed all (slow) tests ;)

**Pull request merged**